data/railsconf/railsconf-2021/videos.yml

```diff
@@ -1173,7 +1301,9 @@
   thumbnail_md: https://i.ytimg.com/vi/ntTT64hK1no/hqdefault.jpg
   thumbnail_lg: https://i.ytimg.com/vi/ntTT64hK1no/sddefault.jpg
   thumbnail_xl: https://i.ytimg.com/vi/ntTT64hK1no/maxresdefault.jpg
-  published_at: "2022-09-28"
+  published_at: "2021-04-12"
+  starts_at: "2021-04-12"
```

Comment:
Isn't the conventstion that you usually use `*_at` only for DateTime and `*_on` or `*_date` for Date?

Reply:
good point will update that

Reply:
actually it was completely wrong in all aspect 
those date belongs to the playlist not videos
fixed in 9843ceb90031065045e9ef93493865fcad297f0e